blueocean-rest-impl/src/test/java/io/jenkins/blueocean/service/embedded/UserSSHKeyTest.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2017, CloudBees, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package io.jenkins.blueocean.service.embedded;
+
+import com.mashape.unirest.http.exceptions.UnirestException;
+import hudson.model.User;
+import java.io.IOException;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Test for User's Jenkins-managed public/private key pair
+ * @author kzantow
+ */
+public class UserSSHKeyTest extends BaseTest {
+
+    public UserSSHKeyTest() {
+        super(true);
+    }
+    
+    @Test
+    public void createPersonalSSHKey() throws IOException, UnirestException {
+        User user = login();
+
+        Map resp = new RequestBuilder(baseUrl)
+                .status(200)
+                .jwtToken(getJwtToken(j.jenkins, user.getId(), user.getId()))
+                .get("/organizations/jenkins/user/publickey").build(Map.class);
```

Comment:
@kzantow Fact that `/organizations/jenkins/users/{user.getId()}/publickey`, better to authorize if logged in user UserImpl.user.getId()  is same as User.current.getId(), if not same then throw `UnauthorizedException`. Something like below for both GET and DELETE implementation. I think this can pose as security hole 🐛 

```
User authenticatedUser =  User.current();
if(authenticatedUser == null) {
    throw new ServiceException.UnauthorizedException("No authenticated user found");
}
if(!user.getId.equals(authenticatedUser.getId()){
    throw new ServiceException.ForbiddenException("Not authorized");
}
```